benchmarks/bench_serve.py

```
@@ -0,0 +1,338 @@
+                        help="Server base URL (default: http://127.0.0.1:30000)")
+    parser.add_argument("--model", default="Qwen3.5-2B",
+                        help="Model name for the API request")
+    parser.add_argument("--max-tokens", type=int, default=128,
+                        help="Max tokens to generate per request (default: 500)")
+    parser.add_argument("--temperature", type=float, default=0.6,
+                        help="Sampling temperature (default: 0.6)")
+    parser.add_argument("--warmup", type=int, default=1,
+                        help="Number of warmup requests before benchmarking (default: 1)")
+    args = parser.parse_args()
+
+    print(f"pymllm Server Benchmark")
```

Ruff (0.15.14)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)